Mage/src/main/java/mage/abilities/common/LinkedSimpleStaticAbility.java

```diff
@@ -0,0 +1,82 @@
+package mage.abilities.common;
+
+import mage.abilities.effects.Effect;
+import mage.constants.Zone;
+import mage.util.CardUtil;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * Warning: please test with a lot of care when using this class for new things.
+ * <p>
+ * A static Ability linked to an Effect.
+ * The parent Ability does take responsability of setting the id for the child.
+ *
+ * @author Susucr
+ */
+public class LinkedSimpleStaticAbility extends SimpleStaticAbility {
```

Comment:
I think `LinkedEffectIdStaticAbility` might be a better name, as linked abilities have a defined rules meaning which is not the same as this.